src/python_interpreter/sysconfig-linux.json

```diff
@@ -302,6 +302,33 @@
       "ext_suffix": ".cpython-311-powerpc64le-linux-gnu.so",
       "abi_tag": "311",
       "pointer_width": 64
+    },
+    {
+      "major": 3,
+      "minor": 7,
+      "abiflags": "",
+      "interpreter": "pypy",
+      "ext_suffix": ".pypy37-pp73-ppc_64-linux-gnu.so",
```

Comment:
Looks like this one should be `.pypy37-pp73-linux-gnu.so`

https://github.com/messense/python-sysconfig-collector/blob/b37307b13aad1ce17eaff033a713e26b4223fd57/sysconfig-linux.json#L311

Fixed in https://github.com/PyO3/maturin/commit/63e8b52d9b2703321b33bb7f545fe8ccf57cee5b

Reply:
Good catch! However, I believe `.pypy37-pp73-ppc_64-linux-gnu.so` is correct and it's just our 3.7 builds at conda-forge are at fault.
Even the `aarch64` one has the architecture part missing:
```sh
podman run --rm --arch=aarch64 --volume=./qemu-aarch64-static:/usr/bin/qemu-aarch64-static quay.io/condaforge/mambaforge sh -c 'mamba create -p/opt/pypy3.7 pypy3.7 && /opt/pypy3.7/bin/python -m sysconfig' 2> /dev/null | grep 'Platform\|EXT_SUFFIX'           [130]
Platform: "linux-aarch64"
	EXT_SUFFIX = ".pypy37-pp73-linux-gnu.so"
```
Also, comparison with other distros:
```sh
podman run --rm --volume=./qemu-ppc64le-static:/usr/bin/qemu-ppc64le-static docker.io/kth5/archpower:latest-ppc64le sh -c 'pacman -Syy --noconfirm pypy3 && pypy3 -m sysconfig' 2> /dev/null | grep 'Platform\|EXT_SUFFIX'
Platform: "linux-ppc64le"
	EXT_SUFFIX = ".pypy39-pp73-ppc_64-linux-gnu.so"
```
```sh
podman run --rm --arch=ppc64le -v ./qemu-ppc64le-static:/usr/bin/qemu-ppc64le-static fedora:36 sh -c 'pkgs="pypy3.7 pypy3.8 pypy3.9" ; dnf install -y ${pkgs} && for py in ${pkgs} ; do ${py} -m sysconfig ; done' 2> /dev/null | grep 'Platform\|EXT_SUFFIX'
Platform: "linux-ppc64le"
	EXT_SUFFIX = ".pypy37-pp73-ppc_64-linux-gnu.so"
Platform: "linux-ppc64le"
	EXT_SUFFIX = ".pypy38-pp73-ppc_64-linux-gnu.so"
Platform: "linux-ppc64le"
	EXT_SUFFIX = ".pypy39-pp73-ppc_64-linux-gnu.so"
```

Shall we use the `fedora:36` (for consistent results for `pypy3.7`/`pypy3.8`/`pypy3.8`) one instead of conda-forge's, maybe?

Reply:
Sorry for this little hiccup. gh-1260 uses sysconfigs from `fedora:36` and should resolve this issue.